src/common/slack.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * This file contains helper methods that post to the CCDL slack channel
+ */
+
+/**
+ * Send data to slack, configured in CCDL channel
+ * @param {object} params Slack webhooks params
+ */
+export async function postToSlack(params) {
+  return await fetch(
+    'https://hooks.slack.com/services/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',
+    {
+      method: 'POST',
+      body: JSON.stringify(params)
+    }
+  );
+}
+
+export async function submitSearchDataRequest(query, values) {
+  let { ip } = await (await fetch('https://api.ipify.org?format=json')).json();
+
+  await postToSlack({
+    attachments: [
+      {
+        fallback: `Missing data for search term '${query}'`,
+        color: '#2eb886',
+        title: `Missing data for search term '${query}'`,
```

Comment:
I haven't kept track of refine.bio code for a while, so this question might be dumb: since `fallback` and `title` are equal, can we keep only one of them?

Reply:
Thanks for looking into this @dongbohu! 

I'n not sure that it can be removed, got the text from one [slack api example](https://api.slack.com/docs/messages/builder?msg=%7B%22attachments%22%3A%5B%7B%22fallback%22%3A%22Required%20plain-text%20summary%20of%20the%20attachment.%22%2C%22color%22%3A%22%2336a64f%22%2C%22pretext%22%3A%22Optional%20text%20that%20appears%20above%20the%20attachment%20block%22%2C%22author_name%22%3A%22Bobby%20Tables%22%2C%22author_link%22%3A%22http%3A%2F%2Fflickr.com%2Fbobby%2F%22%2C%22author_icon%22%3A%22http%3A%2F%2Fflickr.com%2Ficons%2Fbobby.jpg%22%2C%22title%22%3A%22Slack%20API%20Documentation%22%2C%22title_link%22%3A%22https%3A%2F%2Fapi.slack.com%2F%22%2C%22text%22%3A%22Optional%20text%20that%20appears%20within%20the%20attachment%22%2C%22fields%22%3A%5B%7B%22title%22%3A%22Priority%22%2C%22value%22%3A%22High%22%2C%22short%22%3Afalse%7D%5D%2C%22image_url%22%3A%22http%3A%2F%2Fmy-website.com%2Fpath%2Fto%2Fimage.jpg%22%2C%22thumb_url%22%3A%22http%3A%2F%2Fexample.com%2Fpath%2Fto%2Fthumb.png%22%2C%22footer%22%3A%22Slack%20API%22%2C%22footer_icon%22%3A%22https%3A%2F%2Fplatform.slack-edge.com%2Fimg%2Fdefault_application_icon.png%22%2C%22ts%22%3A123456789%7D%5D%7D) where they use both properties but the values are different. Let me test it.

Reply:
Seems they use the `fallback` value for the title of the notification, without it defaults to the organization title. Might be better to keep both.